lib/btc.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from bitcoinrpc.authproxy import AuthServiceProxy, JSONRPCException
+  
+class BtcGRPC:
+    def __init__(self):
```

Comment:
```suggestion
class BtcRPC:
    def __init__(self):
```

It's an RPC connection, grpc is slightly different.